contracts/scripts/util.ts

```diff
@@ -89,9 +89,11 @@ export async function getFacets(diamondAddress: string): Promise<FacetMap> {
 async function startGanache() {
     return new Promise((resolve, reject) => {
         const server = ganache.server({
-            gasPrice: '0x0', // Set gas price to 0
+            miner: { defaultGasPrice: '0x0' },
+            chain: { hardfork: 'berlin' },
+            logging: { quiet: true },
         })
-        server.listen(8545, (err) => {
+        server.listen(18678, (err) => {
```

Comment:
Is this change driven by the previous port not binding successfully to that port? It the new value randomly selected maybe?  Should we maybe move this into a const?

Reply:
thanks for the suggestion - implemented in this commit: c04fabef